ansible/roles/bench_app_service/tasks/main.yml

```diff
@@ -2,6 +2,9 @@
 - name: Install valgrind
   ansible.builtin.apt:
     name: valgrind
+- name: Install cmake
+  ansible.builtin.apt:
+    name: cmake
```

Comment:
Could you add `cmake` to the flake.nix [`native-build-inputs`](https://github.com/aochagavia/rustls-bench-app/blob/a82cb091392f9660c8b61cc1799a6e561fb5ea33/flake.nix#L26C56-L26C56) as well?